frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -118,17 +118,39 @@ export default class FirebaseInteractor {
     }
   }
 
+  /**
+   * update intervention idx number
+   */
+  async updateIntervention(
+    setId: string,
+    wordIdx: number,
+    activityIdx: number
+  ) {
+    let interventions = await this.db
+      .collection("interventions")
+      .doc(setId)
+      .get();
+    let intervention = interventions.ref;
+    intervention.update({
+      wordIdx,
+      activityIdx,
```

Comment:
You do not need to get here. Instead, you can just do `update` right away. Also, the update should be `await`ed so that it waits for the promise to resolve